.changelog/unreleased/bug-fixes/provider/1925-apply-audit-suggestions.md

```diff
@@ -0,0 +1,3 @@
+- Apply audit suggestions that include a bug fix in the way we compute the
```

Comment:
we should avoid making it seem like there was an external audit of PSS (since we just had internal auditors look over it and I don't think the report will be released (?), I would recommend just changing this to
```suggestion
- Fix a bug in the way we compute the
```